i18n/en.yaml

```
@@ -25,6 +25,8 @@ menu.location: Location
 menu.sponsors: Sponsors & Partners
 location_page.heading: Our Location
 location_page.directions_heading: Directions
+location_page.map.frame: Interactive map showing the location of the event
+location_page.map.fullscreen: View larger map of the event location
```

Please check the guidelines ☺️:

```suggestion
location_page.map_frame: Interactive map showing the location of the event
location_page.map_fullscreen: View larger map of the event location
```